README.md

```diff
@@ -2,6 +2,12 @@
 
 Example usage of Nemo automation for a node.js web application (using kraken-js)
 
+## Pre-requisite
+
+It is recommended that you are running `node@v4` or above. If you are using `v0.12` for some reason, note that newer versions of selenium-webdriver 
+use ES6 features. You will need to make sure your node process is started with the `harmony` flag. In general, it's best to use `node@v4` or newer.
+
+**If you plan to run the `wdb-spec.js` test** (it doesn't run by default), you will need to have an appropriate version of the selenium-standalone jar file on your system. Please [see below](#progression-of-automation-tests) for more information.
```

Comment:
There is a link to the main selenium downloads page further down. I don't want to list specific versions because that makes the README out-of-date so quickly.